src/datepicker/datepicker.js

```diff
@@ -268,6 +268,26 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
           });
         }
 
+        // disable navigation according to the range (maxDate and minDate)
+        // compare minDate with the first day of the month
+        var calendarMinDate = new Date(firstDayOfMonth.getTime());
+        calendarMinDate.setHours(0);
+        // if the first day of the month is not greater then minDate there is no point to navigate to a previous month
+        if (ctrl.minDate != null) {
```

Comment:
Is the intention to catch other values than null or also to catch undefined and other falsy values? An explicit check for the desired statement is preferred here for readability